rkt/image_gc.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/spf13/cobra"
+	"github.com/coreos/rkt/common"
+	"github.com/coreos/rkt/pkg/lock"
+	"github.com/coreos/rkt/store"
+)
+
+var (
+	cmdImageGc = &cobra.Command{
+		Use:   "gc",
+		Short: "Garbage collect local store",
+		Run:   runWrapper(runGcImage),
+	}
+)
+
+func init() {
+	cmdImage.AddCommand(cmdImageGc)
+}
+
+func runGcImage(cmd *cobra.Command, args []string) (exit int) {
+	s, err := store.NewStore(globalFlags.Dir)
+	if err != nil {
+		stderr("rkt: cannot open store: %v", err)
+		return 1
+	}
+
+	if err := gcTreeStore(s); err != nil {
+		stderr("rkt: failed to remove unreferenced treeStores: %v", err)
```

Comment:
I think the way it is now is OK: `treestoreID(s)` when functions for getting the IDs fail and `treestore(s)` when we refer to the directory (e.g. removing a treestore).